### PR TITLE
Editorial: Use HTML's definitions of "entry list" and "entry"

### PR DESCRIPTION
**Pull request by andreubotella**
Companion to whatwg/html#7371.

***
<a href="https://whatpr.org/xhr/338.html" title="Last updated on Nov 30, 2021, 8:55 AM UTC (e571c51)">Preview</a> | <a href="https://whatpr.org/xhr/338/88a61d5...e571c51.html" title="Last updated on Nov 30, 2021, 8:55 AM UTC (e571c51)">Diff</a>

### CODE REVIEW

File: xhr.bs [hunk 1]
```diff
@@ -1595,39 +1595,18 @@ interface FormData {
...
+<dfn id=concept-formdata-entry-list export for=FormData>entry list</dfn> (an <a for="/">entry
+list</a>). It is initially empty.
 
-<p>An <dfn id=concept-formdata-entry export for=FormData lt=entry|entries>entry</dfn> consists of a
-<dfn id=concept-formdata-entry-name export for=FormData/entry>name</dfn> and a
-<dfn id=concept-formdata-entry-value export for=FormData/entry>value</dfn>.
+<span id="concept-formdata-entry"></span>
+<span id="concept-formdata-entry-name"></span>
+<span id="concept-formdata-entry-value"></span>
+<span id="create-an-entry"></span>
```
Comment (annevk) [inline xhr.bs:1604]: Put these IDs on the terms in the note below?

File: xhr.bs [hunk 2]
```diff
@@ -1659,8 +1638,8 @@ method steps are:
 <ol>
  <li><p>Let <var>value</var> be <var>value</var> if given; otherwise <var>blobValue</var>.
 
- <li><p>Let <var>entry</var> be the result of <a lt="create an entry">creating an entry</a> with
- <var>name</var>, <var>value</var>, and <var>filename</var> if given.
+ <li><p>Let <var>entry</var> be the result of <a for="entry list" lt="create an entry">creating an
```
Comment (annevk) [inline xhr.bs:1641]: No newline inside `<a>`. You might also be able to drop the `lt` attribute here. I think Bikeshed improved since this was written.

**Review by annevk — APPROVED** [review]

**Comment by andreubotella** [comment]
The updated anchors have now landed in [tabatkins/bikeshed-data](https://github.com/tabatkins/bikeshed-data), so this PR can now be merged.

File: xhr.bs [hunk 3]
```diff
@@ -1595,39 +1595,15 @@ interface FormData {
 </pre>
 
 <p>Each {{FormData}} object has an associated
-<dfn id=concept-formdata-entry-list export for=FormData>entry list</dfn> (a <a for=/>list</a> of
-<a>entries</a>). It is initially the empty list.
+<dfn id=concept-formdata-entry-list export for=FormData>entry list</dfn> (an <a for="/">entry
+list</a>). It is initially empty.
```
Comment (annevk) [inline xhr.bs:1599]:
```suggestion
<dfn id=concept-formdata-entry-list export for=FormData>entry list</dfn> (an 
<a for=/>entry list</a>). It is initially empty.
```

File: xhr.bs [hunk 4]
```diff
@@ -1595,39 +1595,15 @@ interface FormData {
...
- <li><p>Return <var>entry</var>.
-</ol>
+<p class="note">This section used to define
+<a id="concept-formdata-entry" for="entry list">entry</a>, an entry's
+<a id="concept-formdata-entry-name" for="entry list/entry">name</a> and
+<a id="concept-formdata-entry-value" for="entry list/entry">value</a>, and the
+<a id="create-an-entry" for="entry list">create an entry</a> algorithm. These definitions have been
+moved to the HTML Standard. [[HTML]]</p>
 
 <hr>
```
Comment (annevk) [inline xhr.bs:1608]:
```suggestion
```

File: xhr.bs [hunk 5]
```diff
@@ -1595,41 +1595,15 @@ interface FormData {
...
- <li><p>Return <var>entry</var>.
-</ol>
-
-<hr>
+<p class="note">This section used to define
+<a id="concept-formdata-entry" for="entry list">entry</a>, an entry's
+<a id="concept-formdata-entry-name" for="entry list/entry">name</a> and
+<a id="concept-formdata-entry-value" for="entry list/entry">value</a>, and the
+<a id="create-an-entry" for="entry list">create an entry</a> algorithm. These definitions have been
+moved to the HTML Standard. [[HTML]]</p>
```
Comment (annevk) [inline xhr.bs:1606]:
```suggestion
moved to the HTML Standard. [[HTML]]
```

File: xhr.bs [hunk 6]
```diff
@@ -1595,41 +1595,15 @@ interface FormData {
 </pre>
 
 <p>Each {{FormData}} object has an associated
-<dfn id=concept-formdata-entry-list export for=FormData>entry list</dfn> (a <a for=/>list</a> of
-<a>entries</a>). It is initially the empty list.
+<dfn id=concept-formdata-entry-list export for=FormData>entry list</dfn> (an
```
Comment (annevk) [inline xhr.bs:1598]:
```suggestion
<dfn id=concept-formdata-entry-list export for=FormData>entry list</dfn> (an
```

**Comment by annevk** [comment]
Thanks @andreubotella!

**Pull request merged**